stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/AttachmentsPickerDialogStyle.kt

```diff
@@ -74,6 +77,7 @@ public data class AttachmentsPickerDialogStyle(
     val submitAttachmentsButtonIconDrawable: Drawable,
     val attachmentTabToggleButtonStateList: ColorStateList?,
     // Media attachments tab
+    val mediaAttachmentsTabEnabled: Boolean,
```

Comment:
Should we be inserting these values in the list like that? Isn't this a breaking change? cc @adasiewiczr 

Usually we add new items at the bottom when possible, as that way we're not breaking anything for our users.